app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/MainActivity.kt

```diff
@@ -251,7 +251,9 @@ class MainActivity : SimpleActivity() {
                         handlePermission(PERMISSION_READ_CONTACTS) {
                             handleNotificationPermission { granted ->
                                 if (!granted) {
-                                    PermissionRequiredDialog(this, R.string.allow_notifications_incoming_messages)
+                                    PermissionRequiredDialog(this, R.string.allow_notifications_incoming_messages, { openNotificationSettings() }) {
```

Comment:
lets make it clearer by specifying "positiveActionCallback =" and "negativeActionCallback ="

Reply:
updated!